pkgs/top-level/all-packages.nix

```diff
@@ -376,6 +376,8 @@ with pkgs;
 
   circumflex = callPackage ../applications/networking/circumflex { };
 
+  citorn = callPackage ../tools/misc/citron { };
```

Comment:
:)

```suggestion
  citron = callPackage ../tools/misc/citron { };
```

Reply:
Oh my gosh this is an embarrassing one 😳

Reply:
btw. fix for this typo is merged and squashed.